### PR TITLE
Clarified slicing in intro

### PR DESCRIPTION
**Pull request by bw4sz**
Thanks for the note before. I reverted to the repo and added my changes to the .Rmd file, which i then committed. 

I showed how to remove rows and columns using a negative sign, which was a question people had at the WSU workshop when we showed this bit of the lesson.

### CODE REVIEW

File: 01-starting-with-data.Rmd [hunk 1]
```diff
@@ -229,12 +229,38 @@ If we don't include a slice for the rows, R returns all the rows; if we don't in
 If we don't provide a slice for either rows or columns, e.g. `dat[, ]`, R returns the full data frame.
 
 ```{r}
-# All columns from row 5
+# Values from row 5 and all columns
 dat[5, ]
-# All rows from column 16
+# Values from alls rows for column 16
```
Comment (jdblischak) [inline 01-starting-with-data.Rmd]: alls -> all

**Comment by jdblischak** [comment]
Using a negative number in an index is addressed in one of the challenge questions in the lesson. If you've already detailed what it does, it makes the challenge rather uninteresting. Did you have your learners perform that challenge in your workshop? I usually cover negative indices when reviewing the answer to that challenge.

**Comment by bw4sz** [comment]
Hey John,

The main reason i suggested the change was that it was a spontaneous
question from multiple people. Perhaps you are right that the challenge is
alittle preempted, but it was my feeling that if people are wondering
something, its better to have the challenge reinforce something taught,
then be unknown. I understand your point though.

Thanks for maintaining this repo!


On Mon, May 4, 2015 at 8:31 PM, John Blischak <notifications@github.com>
wrote:

> Using a negative number in an index is addressed in one of the challenge
> questions in the lesson. If you've already detailed what it does, it makes
> the challenge rather uninteresting. Did you have your learners perform that
> challenge in your workshop? I usually cover negative indices when reviewing
> the answer to that challenge.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/r-novice-inflammation/pull/84#issuecomment-98927788>
> .
>



-- 
Ben Weinstein
PhD Candidate
Ecology and Evolution
Stony Brook University

http://benweinstein.weebly.com/

File: 01-starting-with-data.Rmd [hunk 2]
```diff
@@ -229,12 +229,38 @@ If we don't include a slice for the rows, R returns all the rows; if we don't in
...
 dat[, 16]
 ```
 
+We can also remove rows and columns using the negative sign.
+
+Recall that our data frame, `dat`, has `r nrow(dat)` rows and `r ncol(dat)` columns.
+
+```{r}
+#remove 5th row
+a5<-dat[-5, ]
```
Comment (jdblischak) [inline 01-starting-with-data.Rmd]: Please surround the assignment operator with spaces, i.e. `a5 <- dat[-5, ]`.

**Comment by jdblischak** [comment]
OK, @bw4sz, I agree that repetition can be a good way to reinforce these new concepts. Please fix the few minor things I pointed out, and then I'll merge this.

**Comment by jdblischak** [comment]
I'm closing this PR. Please feel free to fix the minor changes I detailed and reopen it. Please let me know if you need any help with this.

**Pull request closed without merging**